### PR TITLE
IGAPP-699: Webview crashing the app

### PR DESCRIPTION
**Pull request by klinzo**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

### CODE REVIEW

**Comment by klinzo** [comment]
https://github.com/software-mansion/react-native-screens/issues/105#issuecomment-605369538
https://github.com/react-native-webview/react-native-webview/issues/575#issuecomment-635720390
https://github.com/react-navigation/react-navigation/issues/9067#issuecomment-728095564

**Comment by klinzo** [comment]
we really need e2e tests :thinking:

**Comment by klinzo** [comment]
App crashes only on production builds, with development builds and metro bundler nothing happens.
Please test this and make sure to run `yarn android:integreat:production` to do so.

File: native/src/App.tsx [hunk 1]
```diff
@@ -21,9 +21,7 @@ import SnackbarContainer from './components/SnackbarContainer'
 import NetInfo from '@react-native-community/netinfo'
 import sendTrackingSignal from './services/sendTrackingSignal'
 import useSendOfflineJpalSignals from './hooks/useSendOfflineJpalSignals'
-import { enableScreens } from 'react-native-screens'
 
-enableScreens(true)
```
Comment (maxammann) [inline native/src/App.tsx:26]: I think it would be good tp explicitly state false here. Maybe screens gets turned on by default at some point in time.

**Review by maxammann — APPROVED** [review]

File: release-notes/unreleased/IGAPP-699-webview-crashing.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+issue_key: IGAPP-699
+show_in_stores: true
+platforms:
+  - android
+en: Frequent app crashes fixed.
```
Comment (SteffiStuffel) [inline release-notes/unreleased/IGAPP-699-webview-crashing.yml:5]: Maybe we could express this more positive. Like "increased app stability"? When I would read this in a store I maybe would not download the app, assuming it is very buggy.

**Review by SteffiStuffel — APPROVED** [review]

**Pull request merged**